pkg/firewall/firewall.go

```diff
@@ -70,41 +68,38 @@ type stakeOrActiveKeepPolicy struct {
 func (soakp *stakeOrActiveKeepPolicy) Validate(
 	remotePeerPublicKey *ecdsa.PublicKey,
 ) error {
-	remotePeerNetworkPublicKey := coreKey.NetworkPublic(*remotePeerPublicKey)
-	remotePeerAddress := coreKey.NetworkPubKeyToChainAddress(
-		&remotePeerNetworkPublicKey,
-	)
+	remotePeerOperatorID := soakp.chain.PublicKeyToOperatorID(remotePeerPublicKey)
 
-	logger.Debugf("validating firewall rules for [%v]", remotePeerAddress)
+	logger.Debugf(
+		"validating firewall rules for operator ID [%v]",
+		remotePeerOperatorID,
+	)
 
 	// Validate minimum stake policy. If the remote peer has the minimum stake,
 	// we are fine and we should let to connect.
 	if err := soakp.minimumStakePolicy.Validate(remotePeerPublicKey); err == nil {
 		return nil
 	}
 
-	// Check if the remote peer has authorization on the factory.
-	// The authorization cannot be revoked.
 	// If peer has no authorization on the factory it means it has never
-	// participated in any group selection so there is no chance it can be
 	// a member of any keep.
```

Comment:
Did we remove those comment lines on purpose?

Reply:
(Same caveat as above... this seems likely to have been a merge fumble.)